generators/typescript/sdk/client-class-generator/src/endpoint-request/GeneratedFileUploadEndpointRequest.ts

```diff
@@ -64,7 +64,10 @@ export class GeneratedFileUploadEndpointRequest implements GeneratedEndpointRequ
         this.generatedSdkClientClass = generatedSdkClientClass;
         this.targetRuntime = targetRuntime;
 
-        if (requestBody.properties.some((property) => property.type === "bodyProperty")) {
+        if (
+            requestBody.properties.some((property) => property.type === "bodyProperty") ||
+            endpoint.queryParameters.length > 0
+        ) {
```

Comment:
what does this do?

Reply:
So previously, if there was no bodyParameter, we would never initialize the request parameter or query parameter objects here. This would make it if you had only files as properties and then query parameters, those would get dropped from the request

This coincidentally also impacted this seed test: https://github.com/fern-api/fern/blob/ab/oafileparse/test-definitions/fern/apis/file-upload/definition/service.yml#L34-L51